FX/graysonsolis_Show inserted instrument on currently selected track.lua

```diff
@@ -0,0 +1,74 @@
+-- @description Show Inserted Instrument on Currently Selected Track
+-- @author Grayson Solis
+-- @version 1.0
+-- @link https://graysonsolis.com
+-- @donation https://paypal.me/GrayTunes
+-- @about
+--   This script attempts to open the instrument plugin window (floating) on the currently selected track.
+--   If no instrument is found or its window doesn't open, it falls back to opening the regular FX chain window.
+--
+--   ----------------------------------------------------------------------------------------
+--   USE CASE
+--   ----------------------------------------------------------------------------------------
+--   Useful when:
+--   - You want a one-click way to always pull up the main sound source on a selected track.
+--   - You're working with virtual instruments (VSTi) and need fast access to tweak synths, samplers, etc.
+--
+--
+--   Note: Assign this to a double click mouse modifier for the track! 
+--   - For me, I double click the track and it opens an instrument for me to pick when its empty, and gives me access to the current instrument when I double click!
+--
+--   ----------------------------------------------------------------------------------------
+--   BEHAVIOR
+--   ----------------------------------------------------------------------------------------
+--   - Uses a specific custom command (e.g. from SWS or FX Organizer) to open an instrument’s floating window.
+--   - If the command fails to detect the instrument window, it opens the normal FX chain window instead.
```

Comment:
This script doesn't do that. It opens the FX **browser** window if the first selected track does not contain an instrument FX.